themes/_default.yaml

```diff
@@ -400,7 +400,7 @@ messagebox:
   background-color:
     info: "var(--tx-generic-color-secondary-light)"
     error: "var(--tx-generic-color-negative-light)"
-    success: "var(--tx-generic-color-positive-lighter)"
+    success: "var(--tx-generic-color-positive-light)"
```

Comment:
is this according to our designs?

Reply:
@iwwwi 
well, I've just fixed typo

Reply:
We have this color in `travix-themes` and you shouldn't change the variable. I think you can just set any color for `positive-lighter` in `_default.yaml` file

Reply:
@iwwwi @maximuk 
I've rolled it back, guys!